### ISSUE TITLE
Does the package allow (y_true, y_pred) as input

### ISSUE THREAD

**Issue by pedropalb** [easystats/performance#329]
Hello! 

I am working with a deep neural network model for a regression task. I have the model’s predictions and the ground truths in this a CSV file.

How would I leverage the performance package to evaluate my model’s performance using only the data I have stored in the CSV? Does the package allows as input only the vector pair (y_true, y_pred)? If not, do you plan to implement such a feature? I believe that it would allow to evaluate models coming from all sorts of environments (e.g. Python + TensorFlow in my case) increasing the package generalization.

Thank you for the great work!

**Comment by bwiernik** [comment]
Can you give a simple reproducible example of the input and output and model you are wanting to see?

**Comment by DominiqueMakowski** [comment]
I think @pedropalb is refering to a function that is agnostic to the model, that just takes y_true y_pred as input vectors and compute performance score like accuracy, sensitivity etc, similarly to how it works in other languages

**Comment by strengejacke** [comment]
Currently, I have no clear picture how we could evaluate model performance if only this information is given (maybe manually compute residuals etc., and continue from there...).

**Comment by DominiqueMakowski** [comment]
It's true that the indices I mentioned referred from the ones available from a confusion matrix for a classification model, whereas OP mention a regression model. @pedropalb do you have a more specific idea of indices that could be computed?
